task_manager/api.py

```diff
@@ -46,6 +46,12 @@ def get(self):
         return auth.logout()
 
 
+@api.route('/viewTask')
```

Comment:
could we route this to just /tasks? I think that might be an easier URL to remember
and then individual tasks be /tasks/{taskid}

Reply:
currently, /task gives JSON response. I couldn't combine both JSON and rendering template in a way where JSON could be accessed outside the template.

Reply:
maybe it should be specifically /createTask then that returns the json?

I'm looking at `@task_ns.route('/')` which is /tasks I'm assuming and this seems to be fetching tasks in general, so its what the "get" is suppose to do. What we can probably do is just change the route from /viewTask to `@task_ns.route('/')`  and then remove its definition underneath?

Reply:
Yeah, I can change it to @task_ns.route('/'viewTask). It makes sense. But @task_ns.route('/') routes to /tasks that returns all the tasks in JSON.

Reply:
Oh so is this just the function to get the task while the page /tasks/{taskid} will show it?